elasticluster/share/playbooks/roles/jupyter/tasks/python.yml

```
@@ -54,6 +52,17 @@
     state: '{{ pkg_install_state }}'
   loop: '{{jupyter_additional_os_packages[ansible_os_family]}}'
 
+- name: Install mysql development package.
+  package:
+    name: 'mysqlclient-dev'
```

Shouldn't this be `libmysqlclient-dev`?
```suggestion
    name: 'libmysqlclient-dev'
```